### DISCUSSION TITLE
How to use HRLDAS forcing in LIS ?

### DISCUSSION THREAD

**Discussion by Realone24** [NASA-LIS/LISF#1097] [Q&A]
Dear all,

I am using NOAH-MP model in WRF-Hydro and provide HRLDAS-hr format (High resolution land data assimilation system) as a model forcing. I wonder if I could use those forcing's in NASA LIS with NOAH-MP configuration. 

I tried to figure out using LIS user manual but I could not get it how to provide such options. Please let me know if I can use HRLDAS-hr format forcing in LIS if yes then how to provide such option.

These are the variables which I have is used in NOAH-MP under WRF-Hydro

ncdump -h 2016062203.LDASIN_DOMAIN2

variables:
        float Q2D ;
        float T2D ;
        float PSFC ;
        float SWDOWN ;
        float LWDOWN ;
        float U2D ;
        float V2D ;
        float RAINRATE ;

**Comment by karsenau** [comment]
Hi @Realone24:

Thank you for your interest in LIS and for your helpful question. We currently do not have public support in LIS for HRLDAS-hr forcing data, but we can check with @dmocko and @TimLahmers if anyone is working on this, and get back to you soon.
